ophyd/ophydobj.py

```diff
@@ -33,8 +33,11 @@ class OphydObject:
 
     _default_sub = None
 
-    def __init__(self, *, name=None, parent=None):
+    def __init__(self, *, name=None, parent=None, labels=None):
         super().__init__()
+        if labels is None:
```

Comment:
I would move this above the super call so we can expect this to be available.

Reply:
good idea thanks!